tasks/enable_api_access.yml

```diff
@@ -0,0 +1,55 @@
+---
+# Task file for locating the 'clouds.yaml' file and placing it on the deployment host if necessary.
+
+- name: Register Utility Container
```

Comment:
@johnduarte I tried running without this task and it failed. I'm not sure why my environment doesn't detect the 'utility_container' fact.